Directory.Build.props

```diff
@@ -1,4 +1,8 @@
 ﻿<Project>
+  <Import
```

Comment:
😨

Reply:
I figured this would be something that would be included in KoreBuild no?

Reply:
Well, a part of it. KoreBuild uses the files to pull in settings from the file. This is to get repo projects to also pull in settings